mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -1143,9 +1143,9 @@ class WidgetAdapter(
             }
             with(webView) {
                 adjustForWidgetHeight(widget, 0)
-                loadUrl("about:blank")
```

Comment:
AFAIK this is done, because recycled WebViews would show the previous page for a moment otherwise. Maybe change `shouldOverrideUrlLoading()`, so it never opens the browser if the previous or current page is `about:blank`?

Reply:
Maybe set the WebView to invisible until the loading progress is, say, 50%?

Reply:
Maybe set the webview to invisible for 0% only. It has a progress bar which should be shown when loading.

Reply:
Done.

Reply:
Several WebViews still show the previous page until ~70% progress, so hiding at 0% progress doesn't help. I'd still prefer my suggestion in my first comment here.

Reply:
@maniac103 This comment was resolved, but IMO it's currently still open.